docling/backend/docling_parse_backend.py

```diff
@@ -150,10 +150,11 @@ def __init__(self, path_or_stream: Union[BytesIO, Path]):
         super().__init__(path_or_stream)
         self._pdoc = pdfium.PdfDocument(path_or_stream)
         # Parsing cells with docling_parser call
-        if isinstance(path_or_stream, BytesIO):
-            raise NotImplemented("This backend does not support byte streams yet.")
         parser = pdf_parser()
-        self._parser_doc = parser.find_cells(str(path_or_stream))
+        if isinstance(path_or_stream, BytesIO):
+            self._parser_doc = parser.find_cells_from_bytesio(path_or_stream)
+        else:
```

Comment:
can we add a check that the file exists?

Reply:
not here, this is too internal.
the check if the file exists should be in the `DocumentConversionInput`. It is already in place, https://github.com/DS4SD/docling/blob/main/docling/datamodel/document.py#L99